crates/stackable-operator/src/commons/rbac.rs

```diff
@@ -30,12 +30,18 @@ pub enum Error {
 /// Build RBAC objects for the product workloads.
 /// The `rbac_prefix` is meant to be the product name, for example: zookeeper, airflow, etc.
 /// and it is a assumed that a ClusterRole named `{rbac_prefix}-clusterrole` exists.
+/// 'rbac_prefix' is not used to build the names of the serviceAccount and roleBinding objects,
+/// as this caused problems with multiple clusters of the same product within the same namespace
+/// see <https://stackable.atlassian.net/browse/SUP-148> for more details.
+/// Instead the names for these objects are created by reading the name from the cluster object
+/// and appending [-rolebinding|-serviceaccount] to create unique names instead of using the
+/// same objects for multiple clusters.
```

Comment:
Not a blocker and we can always fix it later, but this feels more like a changelog entry or a local comment than a doccomment to me.

Reply:
that is true indeed